tests/phpunit/tests/l10n/wpTextdomainRegistry.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @group l10n
+ * @group i18n
+ * @ticket 39210
+ *
+ * @coversDefaultClass WP_Textdomain_Registry
+ */
+class Tests_L10n_wpTextdomainRegistry extends WP_UnitTestCase {
+	/**
+	 * @var WP_Textdomain_Registry
+	 */
+	protected $instance;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->instance = new WP_Textdomain_Registry();
+	}
+
+	/**
+	 * @covers ::has
+	 * @covers ::set_default_path
```

Comment:
These all need ticket number?

Reply:
Yep, and test classes don't need a ticket number

Reply:
Easier to write the ticket number once on the class instead of for every single function.

Since ticket numbers are mostly meant for tests covering a bugfix, actually better to just remove it from here entirely.